src/licenses/index.ts

```diff
@@ -0,0 +1,147 @@
+The status of the licenses.
+    * @param {GetLicenseOptions} options.cursor - A unique identifier for the last item of one page to get the next page. This will be either the first or last property from the previous response. No cursor is required on the first request.
+    * @param {GetLicenseOptions} options.take - Default: "20"
+The number of records to take before or after cursor. A positive or negative in the range of -100 to 100. Use a number below 0 to take the previous records before the cursor.
+    * @param {GetLicenseOptions} options.subscriptionUuid - Filters licenses by their subscription. Useful for returning licenses on a per seat subscription.
+    * 
+    * @returns {Promise<GetAllLicensesResponse>} All licenses present on the account
+    */
+    getAll: (options?: GetLicenseOptions) => Promise<GetAllLicensesResponse>,
+    /**
+     *  Get one license
+     * 
+     *  @param {string} licenseUuid - The UUID of the license
+     *  @param {{ expand: string[] }} options
+     *  @param {{ expand: string[] }} options.expand - Specify the resource fields to expand as comma separated values eg `expand=subscription,plan`
+     *
+     * @returns { Promise<License>}
+     */
+    getOne: (licenseUuid: string, options?: { expand: string[] }) => Promise<License>,
+    /**
+     *  Get License's Count
+     * 
+     *  @param {GetLicenseCountInput} options
+     *  @param {GetLicenseCountInput} options.subscriptionUuid - The UUID of the subscription
+     *  @param {GetLicenseCountInput} options.status - The status of the licenses, can be one of: "ACTIVE" "CANCELED" "EVALUATION" "SCHEDULED" "TRIALING" "INACTIVE"
+     *
+     * @returns {Promise<GetLicenseCountResponse>}
+     */
+    getCount: ({ subscriptionUuid, status }: GetLicenseCountInput) => Promise<GetLicenseCountResponse>,
+    /**
+    *  Get Purchasers Licenses
+    * 
+    *  @param {GetPurchasersLicensesInput} purchaserData
+    *  @param {GetPurchasersLicensesInput} purchaserData.purchaser - The purchaser of the licenses
+    *  @param {GetPurchasersLicensesInput} purchaserData.productUuid - The UUID of the product that the licenses are on
+    *  @param {GetPurchasersLicensesInput} purchaserData.options - (Optional) extra options for filtering or additional data
+    *
+    * @returns {Promise<License[]>}
+    */
+    getForPurchaser: (purchaserData: GetPurchasersLicensesInput) => Promise<License[]>,
+    /**
+    *  Get licenses for granteeId
+    * 
+    *  @param {string} granteeId - The granteeId for the licenses
+    *  @param {{ expand: string[] }} options
+    *  @param {{ expand: string[] }} options.expand - Specify the resource fields to expand as comma separated values eg `expand=subscription,plan`
+    *
+    * @returns {Promise<License[]>}
+    */
+    getForGranteeId: ({ granteeId }: { granteeId: string }, options?: { expand?: string[] }) => Promise<License[]>,
```

Comment:
Thought the first parameter which is the UUID, don't have to be passed as an object?